apisix/cli/ngx_tpl.lua

```diff
@@ -809,6 +809,64 @@ http {
             }
         }
 
+        location @disable_proxy_buffering {
```

Comment:
If the HTTP client way works, is it no longer needed?

Reply:
if proxy buffering is enabled, openresty would still wait for buffers to be full before sending response to client, isn't it?

Reply:
confirm that `ngx.flush(true)` is already satisfy for streaming output

```
$ restydoc -s ngx.flush

When "ngx.flush(true)" is called immediately after ngx.print or
       ngx.say, it causes the latter functions to run in synchronous mode.
       This can be particularly useful for streaming output.
```

Reply:
true but this is what I meant: https://docs.api7.ai/hub/proxy-buffering#optional-see-buffering-in-effect

Reply:
No, it's probably a misunderstanding, when I reviewed you hadn't deleted the code for the subrequest, but on commit it had, so GitHub recorded the wrong comment location.
If the subrequests have been cleaned up correctly, I don't see a problem.

Reply:
conclusion: the subrequest block should not be removed now?

Reply:
It should.

Reply:
According to research, `proxy_http_version 1.1;` is SSE friendly, which is the default setting of APISIX.

So `proxy_buffering no;` seems unnecessary.

https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_http_version